src/Helpers/isTruthy.ts

```diff
@@ -0,0 +1 @@
+export const isTruthy = (something: any): boolean => something
```

Comment:
My reasoning here is that I wanted to keep this function as simple as possible, and taking advantage of how JavaScript evaluates expressions, anything can be evaluated to `true` or `false`.

So even though it's implemented as an identity function, the TypeScript signature prevents the caller from actually knowing that it's returning the input — since it's signed as returning a `boolean`, TypeScript won't allow you to treat the result differently.

For example, `isTruthy({ theAnswer: 42 }).theAnswer` would fail to compile.

As such, I think it also doesn't need any unit tests or any extra logic (since it's only using a feature of the language). I think this is just normal use of JS truthy/falsey evaluation.

We could do this too:

```ts
import [ identity } from 'ramda'

export const isTruthy: (something: any) => boolean = identity

// or 

export const isTruthy = identity as (something: any) => boolean
```

If you fear there's room for bugs or it's unintuitive I'd go with the `!!` and add some unit tests.

What do you think?

Reply:
I don't really like for the idea of using types to lie about function logic to provide logic. It's a very simple function so I almost didn't even bother making a comment, but I do see a possible issue if someone attempts to use this function and they pass in a value and expect a boolean back, it might take them a while to debug why they are getting non-boolean somewhere else down the line where they expected a boolean. I think we should go with `!!` and unit tests :)

Reply:
Done!